scripts/connect_bluetooth.py

```diff
@@ -251,6 +256,10 @@ def do_battery(self, *args):
         """Get battery voltage."""
         print(self.proxy.get_battery_voltage())
 
+    def do_diagnostics_matrix(self, *args):
+        """Get diagnostics matrix."""
+        print(self.proxy.get_diagnostics_matrix())
+
```

Comment:
I think we should not have this method. It would print unintelligible characters and the output may be huge.